app/assets/javascripts/global.js

```diff
@@ -29,15 +29,6 @@ $(document).ready(function() {
       format: 'h:mm A'
     });
 
-    //Adds listeners to select inputs to toggle disclaimers on rental schedule page
```

Comment:
it seems like its fine to remove this, let me know if i'm wrong @Cobchise (you wrote it)